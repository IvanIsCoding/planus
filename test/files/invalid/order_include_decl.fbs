table Foo {}
include "/dev/null";