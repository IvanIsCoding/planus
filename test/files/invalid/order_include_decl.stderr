[0m[1m[38;5;9merror[0m[1m: Cannot have includes after the first type declaration[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/order_include_decl.fbs:1:1
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m table Foo {}
  [0m[34mâ”‚[0m [0m[34m------------[0m [0m[34mtype declaration was here[0m
[0m[34m2[0m [0m[34mâ”‚[0m include "/dev/null";
  [0m[34mâ”‚[0m [0m[34m--------------------[0m [0m[34minclude was here[0m

Error: Bailing because of errors
